README.en.md

```diff
@@ -67,7 +68,7 @@ You can use [bebasid batch script](https://github.com/bebasid/bebasid/releases/t
 
 ### <ins>Other</ins>
 
-Please head to [USAGE](https://github.com/bebasid/bebasid/blob/master/dev/readme/USAGE.md) for more usage guidelines.
+Please head to [USAGE](https://github.com/bebasid/bebasid/blob/master/dev/readme/USAGE.md) for more usage methods.
```

Comment:
I think for `more usage methods` needs a verb fix, like `for other methods`.

Reply:
Good suggestion